### DISCUSSION TITLE
AI生技應用有多熱？

### DISCUSSION THREAD

**Discussion by Mitochondrionjessica** [1101-GS4719-C/Python-Community-GS4719-C#1928] [General]
DeepMind在2020年推出蛋白質預測模型AlphaFold，是對單股蛋白質預測的突破性研究。覆蓋範圍從蛋白質擴展到所有與生物相關的分子，並能夠對蛋白質資料庫中幾乎所有分子進行預測，且能達到原子等級的精確度。AlphaFold資料庫也已經成為重要的科學研究資源，到目前為止已經超過190個國家140萬用戶進行存取，同時AlphaFold也被用於加速新型疫苗和藥物的開發。新一代 AlphaFold 模型可以預測蛋白質資料庫（PDB）中幾乎所有分子，模型還可以準確預測配體（與「受體」蛋白結合並導致細胞交流方式發生變化的分子）的結構，以及核酸（包含關鍵遺傳訊息的分子）和轉譯後修飾（蛋白質產生後發生的化學變化）。

相較於過往研究一個蛋白質結構需要以數月至數年，在AI世代僅幾秒鐘的時間。似乎呼應了黃仁勳在2024 年世界政府高峰會（World Governments Summit 2024）發表「計算機時代已逝下一個黃金賽道是生命科學」。其原因是計算機科學年年在軟硬體上都不斷推陳出新，但生命科學的進展卻是寥寥可數。

可惜的是生科產業在台灣一直都不被看好，首先是生物產業的研究需要長達數十年的研究與龐大的資金，然而台灣缺少政策與資金投資在醫藥開發研究的產業，反而多數投資在熱門的半導體產業，進而導致生科人才外流。沒有生物科技相關的政策、資金、技術、與人才之下，產業自然沒沒無聞。早在新冠疫苗時就能看出端睨，台灣的疫苗研發速度較國外緩慢，且安全性也未能通過國際標準，因此國家的醫藥長時間且大量的依賴進口。而今在AI時代下，已開發國家的生技產業是否會直線起飛，我國生技產業落後程度是否更加明顯，是值得思考的問題。

**Comment by wanderer657** [comment]
AlphaFold 蛋白質預測模型的問世確實為生物科技領域帶來了顯著進展，但我們也必須意識到這只是其中的一個方面。除了蛋白質預測外，生物科技還涉及到許多其他領域，例如基因編輯、生物製造等。因此，雖然AlphaFold對蛋白質預測的精確性和速度有著重大影響，但這並不意味著生物科技領域的所有問題都已得到解決。

此外，雖然台灣生物科技產業面臨著一些挑戰，但也存在著一些潛在的優勢和機遇。例如，台灣在醫療保健和科學研究方面擁有豐富的資源和人才，可以作為發展生物科技的基礎。因此，通過加強政策支持、增加投資和加強國際合作，台灣生物科技產業仍然有望實現長期可持續的發展。

總之，雖然AlphaFold是生物科技領域的一個重要里程碑，但我們不能過分樂觀。生物科技的發展還需要從多個方面進行努力，並且台灣在這方面也有著潛力和機遇。

**Comment by 20050201Shawn** [comment]
健康是我們最大的資本，記得前陣子看到一則貼文，在癱瘓者的大腦植入晶片，可透過心靈感應操作3C產品。感覺未來會繼續研發相關科技，使失明者有機會重回視覺、憂鬱症患者改善病情，並應用在各種疾病上，前途相當可觀。

我認為這些例子出發點是好的，即使生命科學玄妙難以捉摸，我們能挖透的知識看似有限，台灣也沒有足夠的舞台發展生物產業，但能實際的幫助到人，仍是再好不過。這項蛋白質預測模型的技術也有很大的應用空間，使我們對生命有嶄新的理解和認識，並改善人們的生活。

**Comment by Eileen050824** [comment]
AlphaFold對我這個生科領域的人來說可謂是迄今為止AI在科學領域最偉大的成就。解開這個胺基酸摺疊與生命意義的謎，不僅具有重大的科學意義，對醫學領域也有極高的價值。恰巧，深度學習技術更為這個問題上提供很好的輔助。我相信AI能扮演大幅提升藥物的研發速度，並降低研發成本的催化劑，在發展台灣生醫產業的同時，幫助患者活得更健康、更長壽。

**Comment by benbebobo149** [comment]
其實不只是生技科學方面，很多領域都有很多商機可以發展。因為前陣子晶片的發展過於迅速，許多的應用都還沒跟上，所以現在正是一個大AI時代，會有大量的資金投入在這些應用裏頭。我對這個趨勢感到興奮，因為這代表著我們身處的時代充滿了機遇和挑戰，我們有機會參與到這些前沿科技的發展中，為社會帶來更多的創新和進步。感謝同學的分享，讓我更加了解了這個趨勢的重要性。

**Comment by yyujimmy** [comment]
就AlphaFold而言，其能夠以前所未有的速度和精度預測蛋白質結構，對於生物科學研究和藥物開發具有巨大的潛力。這標誌著AI技術在生命科學領域的深刻應用，並有望加速相關領域的發展。然而，儘管技術本身極具前景，但這種技術還需要相應的政策、資金和人才支持才能充分發揮其潛力。

至於台灣的生物科技產業，文章提出了一些挑戰和問題。台灣政府和企業相對於半導體等產業更偏好投資，這導致了生物科技產業在資金和政策支持方面的不足。這不僅限制了生物科技產業的發展，也導致了人才的外流。同時，台灣在疫苗研發方面的表現也顯示了生物科技產業的相對滯後。

我認為，台灣作為一個科技先進的國家，應該更加重視生物科技產業的發展。這不僅可以帶來新的經濟增長點，還可以提高台灣在國際上的競爭力。因此，政府和企業應該加大對生物科技產業的支持和投資，並制定相應的政策，以促進生物科技產業的健康發展。

**Comment by KH1126** [comment]
AI在生物技術領域的應用正在變得越來越熱門。這是由於AI的強大能力可以用於加速生物技術的發展。
像是藥物開發， AI可以幫助預測藥物分子的相互作用，加速藥物設計和發現過程，以此減少藥物開發的時間和成本。
